src/csharp/Microsoft.Spark.UnitTest/Sql/TypesTests.cs

```diff
@@ -45,7 +45,25 @@ public void TestArrayType()
             Assert.Equal("integer", arrayType.ElementType.TypeName);
             Assert.True(arrayType.ContainsNull);
         }
-
+        
+        [Fact]
+        public void TestMapType()
+        {
+            var schemaJson =
```

Comment:
Since we don't use `var` unless you are using `new` or casting, can we do `string schemaJson` here and fix other places in this file?

Reply:
Sure, I have changed `var schemaJson` to `string schemaJson` in this file.